kubernetes/gitlab-service.yml

```diff
@@ -5,7 +5,8 @@ metadata:
   labels:
     name: gitlab
 spec:
-  type: LoadBalancer
+  type: ClusterIP
+  externalIPs: [172.17.4.201]
```

Comment:
Yes please.

On Saturday 09 April 2016 05:30 PM, Xueshan Feng wrote:
> In kubernetes/gitlab-service.yml
> <https://github.com/sameersbn/docker-gitlab/pull/660#discussion_r59114270>:
>
>> @@ -5,7 +5,8 @@ metadata:
>>    labels:
>>      name: gitlab
>>  spec:
>> -  type: LoadBalancer
>> +  type: ClusterIP
>> +  externalIPs: [172.17.4.201]
>
> On Apr 9, 2016, at 02:35, Sameer Naik <notifications@github.com> wrote:
> In kubernetes/gitlab-service.yml: > @@ -5,7 +5,8 @@ metadata: > labels:
>  > name: gitlab > spec: > - type: LoadBalancer > + type: ClusterIP > +
> externalIPs: [172.17.4.201] is this line supposed to be here?
> Oh, I use Vagrant to test, no load balancer so I have to give it an
> external IP to connect to it. If one uses LoadBalancer, then no need to
> have it. Should I revert to LoadBalancer spec? Xueshan
> … <#>
> — You are receiving this because you authored the thread. Reply to this
> email directly or view it on GitHub
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sameersbn/docker-gitlab/pull/660/files/3386cd735d9fb12fe0e0efebbc2fae08b6bed7db#r59114270>
>

-- 
Sameer Naik

w: http://www.damagehead.com e: sameer@damagehead.com